Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,32 +32,222 @@ a migration tool. This tool can be used to automatically create a
+`Podman <https://podman.io>`__) needs to be installed on your
 operating system for the tool to work:
 
-..  code-block:: shell
+..  tabs::
+
+    ..  group-tab:: Docker
+
+        ..  code-block:: bash
+
+            docker run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+    .. group-tab:: Podman
 
-    docker run --rm --pull always \
-        -v $(pwd):/project \
-        -it ghcr.io/typo3-documentation/render-guides:latest \
-        migrate Documentation
+        ..  code-block:: bash
+
+            podman run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+
+The last parameter (:bash:`Documentation`) is the name of the directory, where your
+:file:`Settings.cfg` is currently placed in.
 
 Try out the rendering locally
 =============================
 
 Use our Docker container to render your documentation locally. Read more about
 :ref:`local rendering <rendering-docs>`.
 
-Improve your documentation to render without warning
-====================================================
+..  _migrate-detailed-steps:
+
+Further steps to adapt to the new rendering
+===========================================
 
-If you believe you found a bug in the new PHP-based rendering, please open
+You should perform the following tasks to conclude the migration to the
+new rendering tool:
+
+1. Improve your documentation to render without warning
+-------------------------------------------------------
+
+Rendering your documentation should not yield any warnings or errors.
+
+If you get error messages, often they refer to wrong indentation, missing
+interlinks, orphaned files or outdated ReST-identifiers.
+
+If you are unable to address a warning/error with changes in your documentation
+and if you believe you found a bug in the new PHP-based rendering, please open
 an `Issue on GitHub <https://github.com/TYPO3-Documentation/render-guides/issues>`__.
 
-Recommended: Activate automatic testing
-=======================================
+2. Remove outdated files
+-----------------------------------
+
+After you have created the :file:`guides.xml` file, you can remove the old
+:file:`Settings.cfg` file.
+
+You can also delete the files :file:`genindex.rst` which was previously used
+to generate an index.
+
+The :file:`screenshots.json` is also no longer evaluated, so you can remove
+that optional file.
+
+3. Adapt :file:`Includes.rst.txt`
+---------------------------------
+
+The main documentation directory can contain a file :file:`Includes.rst.txt`
+to include any fixed text, which will be placed on every page of your rendered
+documentation.
+
+Previously it was also used to define a list of utilized directives/roles.
+
+You can either safely remove that file, or add your fixed text to it.
```

Comment:
Hint, that when removing the file all references must also be dropped.

Reply:
added a note